src/applications/model/gaming-streaming-server.cc

```diff
@@ -229,7 +243,7 @@ void
 GamingStreamingServer::Send (void)
 {
   NS_LOG_FUNCTION (this);
-  SendPacket (m_pks, m_iat);
+  SendPacket (m_packetSize, m_interArrivalTime);
```

Comment:
if you use vectors directly from this base class you don't need to re-write the this method in the subclasses, only the definition of the RVs